### ISSUE TITLE
The / me command works in the wrong radius if you set the chat radius.

### ISSUE THREAD

**Issue by GrigSolar** [EssentialsX/Essentials#3834]
**Full output of /ess version:**

>.... [16:47:30 INFO]: Server version: 1.16.3-R0.1-SNAPSHOT git-Tuinity-"de1983b" (MC: 1.16.3)
>.... [16:47:30 INFO]: EssentialsX version: 2.18.1.32
>.... [16:47:30 INFO]: PlaceholderAPI version: 2.10.9
>.... [16:47:30 INFO]: LuckPerms version: 5.2.23
>.... [16:47:30 INFO]: Vault version: 1.7.3-b131
>.... [16:47:30 INFO]: EssentialsXChat version: 2.18.1.32
>.... [16:47:30 INFO]: EssentialsXSpawn version: 2.18.1.32
>.... [16:47:30 INFO]: You are running an unsupported server version!

**Server startup log:**
https://gist.github.com/GrigSolar/76be7b0704a0f94a7798766ad8fc43b3

**EssentialsX config:**
https://gist.github.com/GrigSolar/9b32638ae9965445a5d30f5ed68982f8

**Description:**
When I set the chat radius to 40 in the config file, the visibility range of the /me command is significantly reduced. After trying to set different values, I realized that the radius /me is the right size if you set the chat radius to (radius /me * (radius /me + 1)). That is, for /me to work on 40 blocks, you need to set the chat radius to 1640 blocks.

**Steps to reproduce:**
Write command /me.

**Expected behavior:**
Visibility of text from the /me command in a radius equal to the chat radius.

**Comment by triagonal** [comment]
Confirmed. I was able to reproduce on:
```
[21:36:33 INFO]: Server version: 1.16.4-R0.1-SNAPSHOT git-Paper-337 (MC: 1.16.4)
[21:36:33 INFO]: EssentialsX version: 2.19.0-dev+25-001213e
[21:36:33 INFO]: LuckPerms version: 5.2.63
[21:36:33 INFO]: Vault version: 1.7.3-b131
[21:36:33 INFO]: EssentialsXChat version: 2.19.0-dev+25-001213e
```

After setting the chat radius to 20, it seemed /me was only audible in a radius of ~5 blocks. Regular local chat worked as normal.

**Comment by triagonal** [comment]
This has been fixed in the 2.19.0 dev builds (dev+26). Download the latest version from https://essentialsx.net/downloads.html.
